docs/project/get-help.md

```diff
@@ -92,62 +97,122 @@ the easiest way to connect to IRC.
         <td><b>reCAPTCHA</b></td>
         <td>Use the value provided.</td>
       </tr>
-    </table>
+</table>
+
+**Step Three** - Click on the "Connect" button.
+
+The browser connects you to Webchat. You'll see a lot of text. At the bottom of
+the Webchat web page is a command line bar. Just above the command line bar
+a message is shown asking you to register.
+
+ ![Registration needed screen](/project/images/irc_after_login.png)
+
+**Step Four** - Register your nickname by entering the following command in the
+command line bar:
+
+        /msg NickServ REGISTER yourpassword youremail@example.com
+
+ ![Registering screen](/project/images/register_nic.png)
+
+This command line bar is also the entry field that you will use for entering 
+chat messages into IRC chat channels after you have registered and joined a
+chat channel. 
+    
+After entering the REGISTER command, an email is sent to the email address
+that you provided. This email will contain instructions for completing
+your registration.
+
+**Step Five** - Open your email client and look for the email.
+
+ ![Login screen](/project/images/register_email.png)
 
-3. Click "Connect".
+**Step Six** - Back in the browser, complete the registration according to the email
+by entering the following command into the webchat command line bar:
 
-    The system connects you to chat. You'll see a lot of text. At the bottom of
-    the display is a command line. Just above the command line the system asks 
-    you to register.
+        /msg NickServ VERIFY REGISTER yournickname somecode
 
-    ![Login screen](/project/images/irc_after_login.png)
+Your nickname is now registered to chat on freenode.net.
 
+[Jump ahead to join a docker channel and start chatting](#join-a-docker-channel)  
 
-4. In the command line, register your nickname.
+## IRCCloud
 
-        /msg NickServ REGISTER password youremail@example.com
+IRCCloud is a Web Based IRC client service that is hosted in the cloud. 
 
-    ![Login screen](/project/images/register_nic.png)
+> **Note:** 
+> IRCCloud is a Freemium product. 
 
```

Comment:
I would redo this section as steps.  Remember, the folks that read this need *more* help and steps are easier to follow and return to if there are problems.